### PR TITLE
initialHtmlCss

### PR DESCRIPTION
**Pull request by emelysalmeron**
Hi team, 
initialHtmlCss has been added, please review

Thanks

### CODE REVIEW

**Review by inggritenovaleta — COMMENTED** [review]
I run the code and for me it is good. Just a curiosity if we can make like a box to show all the list in the middle. If you run this code, you will see the checkbox would be on the left-end, maybe what about wrapping the list in a div so that it will be shown in the middle? 

image


This I tried with adding div into form
```
    <div class="list" style="display: block;">
    <form>
      <input type="checkbox" id="checkbox1" name="" value="" />
      <label for="checkbox1"> </label><br />
      <input type="checkbox" id="checkbox2" name="" value="" />
      <label for="checkbox2"> </label><br />
      <input type="checkbox" id="checkbox3" name="" value="" />
      <label for="checkbox3"></label><br /><br />
    </form>
  </div>
```

and added css style for .list
```
 .list {
  background: #bdddd7;
  padding: 40px 40px;
  max-width: 30%;
  margin: auto;
  border-radius: 25px;
  box-shadow: 5px 0 5px -5px black, -5px 0 5px -5px black;
  }
```

File: public/css/style.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
+/*
+*
+* ==========================================
+* ==========================================
+*
+*/
+
```
Comment (OksanaShulha) [inline public/css/style.css]: It is better to delete line from 1 to 6 at all :)

Comment (inggritenovaleta) [reply public/css/style.css]: ahh yess true, agreed with Oksana

File: index.html [hunk 2]
```diff
@@ -2,13 +2,62 @@
...
+      <input type="checkbox" id="checkbox1" name="" value="">
+      <label for="checkbox1"> </label><br>
+      <input type="checkbox" id="checkbox2" name="" value="">
+      <label for="checkbox2">  </label><br>
+      <input type="checkbox" id="checkbox3" name="" value="">
+      <label for="checkbox3"></label><br><br>
+    </form>
+
+<!-- Basic Share Links -->
+<h1 class="share-buttons">Share Buttons</h1>
```
Comment (OksanaShulha) [inline index.html]: Those two words **Share Buttons** better to delete (it is enough to have just btns with social links) or to make smaller and change tag to h2.   Only one h1 can exist on one page.

File: public/css/style.css [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  border-top: 4px solid #000;
+  border-bottom: 2px solid #000;
+  padding-bottom: 100px;
+}
+
+/** Social Button CSS **/
+
+.share-buttons {
+  text-align: center;
+}
```
Comment (OksanaShulha) [inline public/css/style.css:52]: Those two words **Share Buttons** better to delete (it is enough just btns with social links) or to make smaller.

Comment (emelysalmeron) [reply public/css/style.css:52]: I changed the class name, thanks for the feedback

File: index.html (hunk 2 above)
Comment (emelysalmeron) [reply index.html]: Alright; updated.

File: index.html [hunk 4]
```diff
@@ -2,13 +2,62 @@
...
+    <br />
+      <code id="warnings" class="warning"></code>
+      <br />
+      <label for=".sort">Sort list by:</label>
+      <select id="sort-type" class="sort">
+        <option value="a">Alphabetical</option>
+        <option value="completed">Completed tasks</option>
+        <option value="uncompleted">Uncompleted tasks</option>
+      </select>
+      <div id="list-container"></div>
```
Comment (OksanaShulha) [inline index.html:31]: div id = "list-container" it is supposed to include a form with checkboxes?)

**Pull request merged**